iceoryx_utils/include/iceoryx_utils/posix_wrapper/timer.hpp

```diff
@@ -42,12 +47,14 @@ enum class TimerError
 
 using namespace iox::units::duration_literals;
 
+
 /// @brief Interface for timers on POSIX operating systems
 /// @note Can't be copied or moved as operating system has a pointer to this object. It needs to be ensured that this
 /// object lives longer than timeToWait, otherwise the operating system will unregister the timer
+/// @concurrent not thread safe
 ///
 /// @code
-///     posix::Timer TiborTheTimer(100_ms, [&]() { fooBar++; });
+///     posix::Timer& TiborTheTimer = posix::Timer::getInstance(100_ms, [&]() { fooBar++; });
```

Comment:
`posix::Timer::getInstance()` is not available. Could you update the example?

Reply:
Done!